README.md

```diff
@@ -16,6 +16,14 @@ npm install --save react-refetch
 
 This assumes that you’re using [npm](http://npmjs.com/) package manager with a module bundler like [Webpack](http://webpack.github.io) or [Browserify](http://browserify.org/) to consume [CommonJS modules](http://webpack.github.io/docs/commonjs.html).
 
+The following ES6 functions are required:
+
+- [`Object.assign`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/assign)
+- [`Promise`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise)
+- [`fetch`](https://developer.mozilla.org/en-US/docs/Web/API/GlobalFetch) (This was previously included in builds, but isn't anymore.)
```

Comment:
This was a peer dependency, but not actually included. I wonder if it would be good to add recommended polyfills for these. Don't block on this. We can always add something later.